services/report/__init__.py

```diff
@@ -662,7 +663,9 @@ async def _possibly_shift_carryforward_report(
                 )
             return carryforward_report
 
-    async def create_new_report_for_commit(self, commit: Commit) -> Report:
+    async def create_new_report_for_commit(
+        self, commit: Commit, report_code: str = None
```

Comment:
Is `report_code` needed any more?  I think that was so you knew where to read from storage.

Reply:
You are correct